health_monitoring/test_health_monitor.py

```diff
@@ -340,3 +342,57 @@ def test_smoke(self):
         self.assertEqual(stats.health_statuses[5], 3)
         # server_failover_http works too for health monitoring
         self.assertEqual(stats.health_statuses[404], 1)
+
+
+class TestHealthMonitorForDisabledServer(tester.TempestaTest):
+    tempesta = {
+        "config": """
+            listen 80;
+            health_check h_monitor1 {
+                request "GET / HTTP/1.1\r\n\r\n";
+                request_url "/";
+                resp_code   200;
+                resp_crc32  auto;
+                timeout     1;
+            }
+
+            srv_group srv_grp1 {
+                server ${server_ip}:8000;
+                health h_monitor1;
+            }
+
+            vhost srv_grp1{
+                proxy_pass srv_grp1;
+            }
+
+            http_chain {
+                -> srv_grp1;
+            }
+        """
+    }
+
+    backends = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+        },
+    ]
+
+    clients = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "addr": "${tempesta_ip}",
+            "port": "80",
+        }
+    ]
+
+    def test(self):
+        # This test reproduce crash from #2066 issue in Tempesta FW
```

Comment:
Please update this comment. Why does this test trigger a kernel BUG? 2066 issue does not have description, only trace. I don't understand why we use `drop_conn_when_receiving_data` but don't send data?